react/components/admin/pages/Form/index.tsx

```diff
@@ -78,15 +79,13 @@ class FormContainer extends Component<Props, State> {
     const { declarer } = props.initialData || { declarer: null }
     const isNew = isNewRoute(props.initialData)
 
-    const isDeletable = !declarer && !isNew
-    const isInfoEditable = !declarer || isNew
-
     this.state = {
       data: props.initialData,
       formErrors: {},
-      isDeletable,
-      isInfoEditable,
+      isDeletable: !declarer && !isNew,
+      isInfoEditable: !declarer || isNew,
       isLoading: false,
+      isNew,
```

Comment:
I don't think `isNew` should be a state variable, since it always stays the same; I would move it to a private attribute.

Reply:
Agreed!